doc/src/pair_lepton.rst

```diff
@@ -0,0 +1,125 @@
+.. index:: pair_style lepton
+.. index:: pair_style lepton/omp
+
+pair_style lepton command
+=========================
+
+Accelerator Variants: *lepton/omp*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style style args
+
+* style = *lepton*
+* args = list of arguments for a particular style
+
+.. parsed-literal::
+
+    *lepton* args = cutoff
+      cutoff = global cutoff for the interactions (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lepton 2.5
+
+   pair_coeff  * *  "k*((r-r0)^2*step(r0-r)); k=200; r0=1.5" 2.0
+   pair_coeff  1 2  "4.0*eps*((sig/r)^12 - (sig/r)^6);eps=1.0;sig=1.0" 1.12246204830937
+   pair_coeff  2 2  "eps*(2.0*(sig/r)^9 - 3.0*(sig/r)^6);eps=1.0;sig=1.0"
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Pair style *lepton* computes spherical pairwise interactions based on
+evaluating strings between neighboring atoms within the given cutoff.
+The potential function must be provided as an expression string using
+"r" as the distance variable. For example `"200.0*(r-1.5)^2"` represents
```

Comment:
.. as the distance variable.  Note that additional constants in the expression can be defined in the string using semicolon separators; see the examples above.   For example "200 ...
(maybe have two versions of the same string with and without the additional constants)

Reply:
done